### PR TITLE
Allow to set multiple assignees on MRs

### PR DESCRIPTION
**Pull request by mvisonneau**
This feature is now available from starter/bronze EE: https://gitlab.com/gitlab-org/gitlab-ce/issues/41299

https://docs.gitlab.com/ee/api/merge_requests.html#create-mr
https://docs.gitlab.com/ee/api/merge_requests.html#update-mr

### CODE REVIEW

File: merge_requests.go [hunk 1]
```diff
@@ -476,6 +476,7 @@ type CreateMergeRequestOptions struct {
 	TargetBranch       *string `url:"target_branch,omitempty" json:"target_branch,omitempty"`
 	Labels             Labels  `url:"labels,comma,omitempty" json:"labels,omitempty"`
 	AssigneeID         *int    `url:"assignee_id,omitempty" json:"assignee_id,omitempty"`
+	AssigneeIDs        []*int  `url:"assignee_ids,omitempty" json:"assignee_ids,omitempty"`
```
Comment (svanharmelen) [inline merge_requests.go]:
```suggestion
	AssigneeIDs        []int  `url:"assignee_ids,omitempty" json:"assignee_ids,omitempty"`
```

File: merge_requests.go [hunk 2]
```diff
@@ -518,6 +519,7 @@ type UpdateMergeRequestOptions struct {
 	Description        *string `url:"description,omitempty" json:"description,omitempty"`
 	TargetBranch       *string `url:"target_branch,omitempty" json:"target_branch,omitempty"`
 	AssigneeID         *int    `url:"assignee_id,omitempty" json:"assignee_id,omitempty"`
+	AssigneeIDs        []*int  `url:"assignee_ids,omitempty" json:"assignee_ids,omitempty"`
```
Comment (svanharmelen) [inline merge_requests.go]:
```suggestion
	AssigneeIDs        []int  `url:"assignee_ids,omitempty" json:"assignee_ids,omitempty"`
```

File: merge_requests.go (hunk 1 above)
Comment (mvisonneau) [reply merge_requests.go]: 👌

File: merge_requests.go (hunk 2 above)
Comment (mvisonneau) [reply merge_requests.go]: 👌

**Comment by mvisonneau** [comment]
Thanks for the review @svanharmelen! I removed the pointers and rebased 👍

**Comment by svanharmelen** [comment]
It still looks the same, did you forget to commit you changes?

**Comment by mvisonneau** [comment]
🤦‍♂ indeed, not sure what happene. It should be fine now!

**Pull request merged**